packages/block-library/src/gallery/edit.js

```diff
@@ -110,6 +120,20 @@ function GalleryEdit( props ) {
 		onFocus,
 	} = props;
 
+	const [ blockSetting ] = useSettings( 'blocks' );
+
+	const lightboxSetting = blockSetting.hasOwnProperty( 'core/image' )
+		? blockSetting[ 'core/image' ]?.lightbox
+		: false;
```

Comment:
This should be easier to write if you specify the path directly:

```javascript
const lightboxSetting = useSettings( 'blocks.core/image.lightbox' );
```

@youknowriad We need to get the settings of the Image block from the Gallery block. Are there any other suitable APIs?

Reply:
Not that I know of, but we might want to provide one (like we do for the global styles UI) where we can pass the block name.